packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -101,20 +105,22 @@ class GoTrueClient {
     GotrueAsyncStorage? asyncStorage,
     AuthFlowType flowType = AuthFlowType.pkce,
   })  : _url = url ?? Constants.defaultGotrueUrl,
-        _headers = headers ?? {},
+        _headers = {
```

Comment:
I've noticed a bug here, which has nothing to do with logging. The default headers weren't used for the non admin api calls. So the client version header was missing.